vpd-tool/include/vpd_tool.hpp

```diff
@@ -74,5 +74,22 @@ class VpdTool
      * @return On success returns 0, otherwise returns -1.
      */
     int dumpObject(const std::string& i_fruPath) const noexcept;
+
+    /**
+     * @brief Reset specific keywords on System VPD to default value.
+     *
+     * This API resets specific System VPD keywords to default value. The
+     * keyword values are reset on:
+     * 1. Primary EEPROM path.
+     * 2. Secondary EEPROM path.
+     * 3. D-Bus cache.
```

Comment:
what about the backup path?

Reply:
I see it has been updated. Mark it as resolved.